spec/support/feature_spec_helper.rb

```diff
@@ -1,6 +1,4 @@
 module FeatureSpecHelper
-  # requires IntegrationSpecHelper
```

Comment:
Well, this is a mixin that requires another mixin, neither of which are included in all specs.